spec/lib/govuk_elements_form_builder/form_builder_spec.rb

```diff
@@ -48,6 +48,19 @@ def size(method, size)
       ]
     end
 
+    it 'supports attributes defined as a string' do
+      output = builder.send method, 'name', class: 'custom-class'
+
+      expect_equal output, [
+        '<div class="form-group">',
+        '<label class="form-label" for="person_name">',
+        'Full name',
+        '</label>',
+        %'<#{element_for(method)} class="form-control custom-class" #{type_for(method, type)}name="person[name]" id="person_name" />',
+        '</div>'
+      ]
```

Comment:
It would be nice if the `custom-class` here weren't buried in the middle of a sea of markup. Maybe it'd be worth refactoring this out to an `expected_form_group` that accepts parameters, or something like that?